selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/answer/AnswerLocator.java

```diff
@@ -344,12 +349,10 @@
   public static final String ELEMENT_QUESTION_ANSWER                               =
                                                      "//*[@class='rightContent']//*[text()='$question']//ancestor::*[@class='rightContent']//*[@class='uiIconAnsAnswer uiIconLightGray']";
 
-  public static final String ELEMENT_QUESTION_EDIT                                 =
-                                                   "//*[@class='rightContent']//*[text()='$question']//ancestor::*[@class='rightContent']//*[@class='uiIconEdit uiIconLightGray']";
-
-  public static final String ELEMENT_QUESTION_DELETE                               =
-                                                     "//*[@class='rightContent']//*[text()='$question']//ancestor::*[@class='rightContent']//*[@class='uiIconTrash uiIconLightGray']";
-
+  public static final SelenideElement ELEMENT_QUESTION_EDIT                                 =
+          $(byId("UIQuestions")).find(byClassName("uiIconEdit "));
+  public static final SelenideElement ELEMENT_QUESTION_DELETE                               =
+          $(byId("UIQuestions")).find(byClassName("uiIconTrash  "));
```

Comment:
Is the trailing space in "uiIconEdit " and "uiIconTrash  " ? It should probably be removed, no ?